cmd/iam_apikey_list.go

```diff
@@ -0,0 +1,52 @@
+package cmd
+
+import (
+	"strings"
+
+	"github.com/exoscale/egoscale"
+	"github.com/spf13/cobra"
+)
+
+type apiKeyItem struct {
+	Description string              `json:"description"`
+	Key         string              `json:"key"`
+	Operations  string              `json:"operations,omitempty"`
+	Type        egoscale.APIKeyType `json:"type"`
```

Comment:
Avoid exposing low-level types in the custom output structure, prefer `string` in this case.